### PR TITLE
create product

### PR DESCRIPTION
**Pull request by thangtv-1934**
==> Title: Create product

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/13578

### CODE REVIEW

**Comment by thangtv-1934** [comment]
ready

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  before_action :logged_in_user, only: %i(new create edit)
+
+  def new; end
+
+  def create
+    @product = Product.new product_params
+    if @product.save
+      flash[:success] = t "product.create_success"
+      redirect_to stores_url
+    end
```
Comment (huuphuc93) [inline app/controllers/products_controller.rb]: else đâu em

**Comment by thangtv-1934** [comment]
ready

File: app/controllers/products_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  def new; end
+
+  def create
+    @product = Product.new product_params
+    if @product.save
+      flash[:success] = t "product.create_success"
+      redirect_to stores_url
+    else
+      render :create
```
Comment (huuphuc93) [inline app/controllers/products_controller.rb]:
```suggestion
      render :new
```

**Comment by thangtv-1934** [comment]
ready

**Pull request merged**